pkgs/by-name/ry/ryujinx/package.nix

```diff
@@ -98,7 +98,7 @@ buildDotnetModule rec {
     install -D ../misc/Logo.svg $out/share/icons/hicolor/scalable/apps/Ryujinx.svg
 
     substituteInPlace $out/share/applications/Ryujinx.desktop \
-      --replace "Ryujinx %f" "$out/bin/Ryujinx %f"
+      --replace "Ryujinx.sh %f" "$out/bin/Ryujinx.sh %f"
```

Comment:
Wait, does `$out/bin/Ryujinx.sh` exist?

Reply:
Yes. 
https://github.com/Ryujinx/Ryujinx/blob/3924bd1a4364455ab8a5747e3cb0b3000dbaa589/distribution/linux/Ryujinx.desktop#L6

Reply:
I mean, after building the package, what are the files in the `result/bin` folder? Did you see a `Ryujinx.sh`?

Reply:
Ryujinx.sh is a wrapper script. We might want to add it (it sets an environment variable, see [the source](https://github.com/Ryujinx/Ryujinx/blob/master/distribution/linux/Ryujinx.sh)) but don't currently

Reply:
Ah, missed it. Added!
```
❯ ls /nix/store/kaxzr63y8dk52pwhqfxfvpnsk4rbf70d-ryujinx-1.1.1217/bin
ryujinx  Ryujinx  Ryujinx.Headless.SDL2  Ryujinx.sh
```